rct229/rulesets/ashrae9012019/section16/section16rule1.py

```diff
@@ -0,0 +1,121 @@
+            id="16-1",
+            description="The elevator peak motor power shall be calculated according to the equation in Table G3.1-16.",
+            ruleset_section_title="Elevators",
+            standard_section="Section G3.1",
+            is_primary_rule=True,
+            rmd_context="ruleset_model_descriptions/0",
+            list_path="ruleset_model_descriptions[0].buildings[*].elevators[*]",
+        )
+
+    def is_applicable(self, context, data=None):
+        rmd_p = context.PROPOSED
+
+        return (
+            len(
+                find_all(
+                    "$.ruleset_model_descriptions[0].buildings[*].elevators", rmd_p
+                )
+            )
+            > 0
+        )
+
+    class ElevatorRule(RuleDefinitionBase):
+        def __init__(self):
+            super(Section16Rule1.ElevatorRule, self).__init__(
+                rmds_used=produce_ruleset_model_instance(
+                    USER=False,
+                    BASELINE_0=True,
+                    PROPOSED=False,
+                ),
+            )
+
+        def get_calc_vals(self, context, data=None):
+            elevator_b = context.BASELINE_0
+
+            total_floors_served_b = elevator_b.get("number_of_floors_served")
+            elevator_motor_power_b = elevator_b.get("motor_power")
+            elevator_cab_weight_b = elevator_b.get("cab_weight")
+            elevator_cab_counterweight_b = elevator_b.get("cab_counterweight")
+            elevator_design_load_b = elevator_b.get("design_load")
+            elevator_speed_b = elevator_b.get("speed")
+
+            has_undetermined = any(
+                [
+                    param is None
+                    for param in (
+                        total_floors_served_b,
+                        elevator_motor_power_b,
+                        elevator_cab_weight_b,
+                        elevator_cab_counterweight_b,
+                        elevator_design_load_b,
+                        elevator_speed_b,
+                    )
+                ]
+            )
+
+            elevator_mechanical_efficiency_b = table_G3_9_2_lookup(
+                total_floors_served_b
+            )["mechanical_efficiency"]
+            motor_brake_horsepower_b = (
+                (
+                    elevator_cab_weight_b
+                    + elevator_design_load_b
+                    - elevator_cab_counterweight_b
+                )
+                * elevator_speed_b
+                / 33000
+                / elevator_mechanical_efficiency_b
+            )
+            elevator_motor_efficiency_b = (
+                table_G3_9_1_lookup(motor_brake_horsepower_b)["motor_efficiency"]
+                if total_floors_served_b > 4
+                else table_G3_9_3_lookup(motor_brake_horsepower_b)["motor_efficiency"]
+            )
+            expected_peak_motor_power_b = (
+                motor_brake_horsepower_b * 746 / elevator_motor_efficiency_b
+            )
```

Comment:
Add comments here to explain where the 33,000 and 746 are coming from.

Reply:
Addressed.